packages/components/grid/src/grid.ts

```diff
@@ -440,6 +456,7 @@ export class Grid<T = any> extends ScopedElementsMixin(LitElement) {
         class=${classMap({ active })}
         part=${parts.join(' ')}
         index=${index}
+        aria-rowindex=${index}
```

Comment:
Should grid also include `aria-posinset` and `aria-setsize` next to `aria-rowindex`? I’ve added the first 2 to combobox. See https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-posinset

Reply:
Never mind the above, but do plz add https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-rowcount

Reply:
Which is already in the DOM :)

Reply:
when the role is table or grid instead of list you need to use `aria-rowcount` and `aria-rowindex`